code/stable/dblpendulum/SRS/PDF/Makefile

```diff
@@ -6,4 +6,4 @@ DblPendulum_SRS.pdf: DblPendulum_SRS.tex
 	lualatex $(TEXFLAGS) DblPendulum_SRS
 	lualatex $(TEXFLAGS) DblPendulum_SRS
 
-.PHONY:
+.PHONY: srs
```

Comment:
Is this a desired change?

Reply:
I'm pretty sure that before #3499 was merged, this line of code existed but then was removed as part of #3499 as it required me to trim out the beginning of `.PHONY` to ensure that `.PHONY` was proper in the `Makefile`s generated with other code files.

I am not sure of the purpose it serves and whether or not it is truly required. However, this change just reinstates the `srs`  word that was lost in #3499.